common/testutil/testutil.go

```diff
@@ -171,3 +174,19 @@ func MergeOplogStreams(input [][]db.Oplog) []db.Oplog {
 
 	return ops
 }
+
+func MakeTempDir(t *testing.T) (string, func()) {
+	require := require.New(t)
+
+	dir, err := ioutil.TempDir("", "mongo-tools-test")
+	require.NoError(err, "can create temp directory")
+	cleanup := func() {
+		if os.Getenv("TOOLS_TESTING_NO_CLEANUP") == "" {
```

Comment:
Let's document this env variable in CONTRIBUTING.md

Reply:
Done. I also added go docs for the `MakeTempDir` func.